samples/MusicStore/ForTesting/Mocks/StartupOpenIdConnectTesting.cs

```diff
@@ -113,6 +115,15 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app)
         {
+            var supportedCultures = new[] { new CultureInfo("en-US") };
+
+            app.UseRequestLocalization(new RequestLocalizationOptions
```

Comment:
Could you add a comment stating why you're forcing `en-US`? Would be helpful for posterity.